CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# [2.0.0](https://github.com/princjef/irontask/compare/v1.5.0...v2.0.0) (2024-3-29)
```

Comment:
revert this file and the other version change stuff you did. the release management tools will do that automatically